test/client_service_base.test.js

```diff
@@ -44,935 +40,857 @@ const clientProperties = {
-                {}, // service
-                null, // protobuf
-                {}, // properties
-            );
-            await expect(
-                clientServiceBase.registerCertificate(),
-            ).to.be.rejected;
-          },
-      );
-      it('should work as expected', async () => {
-        // prepare
-        const mockedCertificateRegistrationRequest = {
-          setCertHolderId: function() {},
-          setCertVersion: function() {},
-          setCertPem: function() {},
-        };
-        const mockedProtobuf = {
-          CertificateRegistrationRequest: function() {
-            return mockedCertificateRegistrationRequest;
-          },
-        };
-        const clientServiceBase = new ClientServiceBase(
-            {
-              ledgerPrivileged: {
-                registerCert: (_, __, callback) => {
-                  const mockedResponse = {
-                    toObject: () => ({}),
-                  };
-                  callback(null, mockedResponse);
-                },
-              },
+// afterEach(function() {
+//   sinon.restore();
+// });
+
+describe('registerCertificate', () => {
+  test('should throw error when some properties are not provied', async () => {
+    const clientServiceBase = new ClientServiceBase(
+        {}, // service
+        null, // protobuf
+        {}, // properties
+    );
+    await expect(
+        clientServiceBase.registerCertificate(),
+    ).rejects.toThrowError();
+  });
+
+  test('should work as expected', async () => {
+    // prepare
+    const mockedCertificateRegistrationRequest = {
+      setCertHolderId: function() {},
+      setCertVersion: function() {},
+      setCertPem: function() {},
+    };
+    const mockedProtobuf = {
+      CertificateRegistrationRequest: function() {
+        return mockedCertificateRegistrationRequest;
+      },
+    };
+    const clientServiceBase = new ClientServiceBase(
+        {
+          ledgerPrivileged: {
+            registerCert: (_, __, callback) => {
+              const mockedResponse = {
+                toObject: () => ({}),
+              };
+              callback(null, mockedResponse);
             },
-            mockedProtobuf,
-            clientProperties,
-        );
-        const mockSpySetCertHolderId = sinon.spy(
-            mockedCertificateRegistrationRequest,
-            'setCertHolderId',
-        );
-        const mockSpySetCertVersion = sinon.spy(
-            mockedCertificateRegistrationRequest,
-            'setCertVersion',
-        );
-        const mockSpySetCertPem = sinon.spy(
-            mockedCertificateRegistrationRequest,
-            'setCertPem',
-        );
-
-        // act
-        const response = await clientServiceBase.registerCertificate();
-
-        // assert
-        assert(
-            mockSpySetCertHolderId.calledWithExactly(
-                clientProperties['scalar.dl.client.cert_holder_id'],
-            ),
-        );
```

Comment:
`assest + calledWithExactly` is replaced by `expect + toBeCalledWith` in Jest

before:
```
        assert(
              mockSpySetCertHolderId.calledWithExactly(
                  clientProperties['scalar.dl.client.cert_holder_id'],
              ),
          );
```

after:
```
    expect(mockSpySetCertHolderId).toBeCalledWith(
         clientProperties['scalar.dl.client.cert_holder_id'],
     );
```